src/screens/ProfileScreen/index.js

```diff
@@ -59,11 +59,12 @@ import {
 import dimen from '../../utils/dimen';
 import {fonts} from '../../utils/fonts';
 import {useUpdateClientGetstreamHook} from '../../utils/getstream/ClientGetStram';
+import ImageCompressionUtils from '../../utils/image/compress';
 import {linkContextScreenParamBuilder} from '../../utils/navigation/paramBuilder';
 import {requestCameraPermission, requestExternalStoragePermission} from '../../utils/permission';
-import {COLORS} from '../../utils/theme';
 import ShareUtils from '../../utils/share';
 import StorageUtils from '../../utils/storage';
+import {COLORS} from '../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `ImageCompressionUtils` from `../../utils/image/compress` has been moved. Similar to the previous comment, ensure that this utility is not used before its import.
```diff
- import ImageCompressionUtils from '../../utils/image/compress';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->